smarts/core/agent_interface.py

```diff
@@ -172,6 +172,16 @@ class AgentsAliveDoneCriteria:
     """
 
 
+@dataclass(frozen=True)
+class EventConfiguration:
+    """Configure the conditions in which an Event is triggered."""
+
+    not_moving_time: int = 60
+    """How long in seconds to check the agent after which it is considered not moving"""
+    not_moving_distance: int = 1
+    """How many meters to check for the agent to move after which it is considered not moving"""
```

Comment:
I believe these values should be `float`.

```suggestion
    not_moving_time: float = 60
    """How long in seconds to check the agent after which it is considered not moving"""
    not_moving_distance: float = 1
    """How many meters the agent's actor has to move under which the agent is considered not moving"""
```